lib/Model/events.js

```diff
@@ -442,6 +442,7 @@ ChangeEvent.prototype._getArgs = function() {
 
 function LoadEvent(value, passed) {
   this.value = value;
```

Comment:
Might be good to add inline comments on the "internal" names, indicating that people should prefer using the "external" documented names like `document`, "removed", etc.

Reply:
I think perhaps we need to revamp our docs to refer to using `value` instead, as that seems more standard / predictable. I think that could be a separate discussion. I will merge this for now, but can always add a comment a bit later once we've settled on a good approach.